### DISCUSSION TITLE
Cursor rendering behind guides

### DISCUSSION THREAD

**Discussion by n-kort** [lukas-reineke/indent-blankline.nvim#847] [Q&A] [answered]
The cursor is somehow rendered behind the blanklines, which means it is concealed in insert mode.

https://github.com/lukas-reineke/indent-blankline.nvim/assets/1377218/13dc3f10-d2de-4dfb-bec7-c5e75ff116df

I use iTerm and the onedark.nvim theme, configuration is otherwise all defaults as far as I can see. Is there some way to change the opacity of these lines or should I be looking at other settings to fix this?

**Comment by lukas-reineke** [comment, marked as answer]
The lines don't have transparency, and the cursor cannot be rendered behind them.
This is a problem with your terminal, I don't think I can help you with this.
